cmd/junitreport/pkg/api/util/time.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"encoding/xml"
+	"time"
+)
+
+// Duration is an alias for time.Duration, because time.Duration does not implement
+// XML marshalling functions, so when a time.Duration is marshalled into XML, it is
+// represented as the unit value of nanoseconds in the duration. This ugly hack is
+// the only way to make the marshalled value useable.
+type Duration time.Duration
```

Comment:
@liggitt  @deads2k Very unhappy about this. If anyone knows of a better way, I'm happy to use it. I do *not* want to store times as strings internally.